tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -686,6 +686,39 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		err = oa.WaitForTidbClusterReady(tc, 30*time.Minute, 15*time.Second)
 		framework.ExpectNoError(err, "Expected TiDB cluster scaled in and ready")
 	})
+	ginkgo.It("TiDBInitializer: Test managing tidb initializer job with TidbInitializer CRD", func() {
+		cluster := newTidbClusterConfig(e2econfig.TestConfig, ns, "init", "admin", "")
+		cluster.Resources["pd.replicas"] = "1"
+		cluster.Resources["tikv.replicas"] = "1"
+		cluster.Resources["tidb.replicas"] = "1"
+		oa.DeployTidbClusterForInitOrDie(&cluster)
```

Comment:
I don't think it's necessary to add any complex methods to `operatorActions`. Right now, it's ok to deploy tidb cluster with plain Kubernetes client.  Here is an example to deploy a tidb cluster and wait for it to be ready. https://github.com/pingcap/tidb-operator/blob/master/tests/e2e/tidbcluster/tidbcluster.go#L654-L688

If you want to update tidb cluster CRD object, you can modify its fixture function or update before creating.